forte/processors/base/data_selector_for_da.py

```diff
@@ -31,19 +31,19 @@
 
 
 __all__ = [
-    "BaseElasticSearchDataSelector",
-    "RandomDataSelector",
-    "QueryDataSelector",
+    "BaseElasticSearchDataSelectorForDA",
+    "RandomDataSelectorForDA",
+    "QueryDataSelectorForDA",
 ]
 
 
-class BaseDataSelector(PackReader):
+class BaseDataSelectorForDA(PackReader):
```

Comment:
I don't think `DataSelector have to "searches from an indexer", it just return data from a document set. So you can change the docstring to make it more general